src/features/applicationView/__tests__/__snapshots__/ApplicationViewContainer.test.tsx.snap

```diff
@@ -34,6 +34,17 @@ exports[`ApplicationViewContainer renders normally 1`] = `
       >
         ASIAKASTIEDOT
       </span>
+      <div
```

Comment:
Hmm, this is pretty big snapshot file (over 1000 lines?). Consider if this is needed at all. Does it give you confidence that the application works correctly or is it a burden to check and update?

Reply:
Good point, removed them in favor of checking that `ApplicationView` is defined